tests/travis/install-db2.sh

```diff
@@ -4,19 +4,20 @@ set -ex
 
 echo Setting up IBM DB2
 
-sudo docker pull ibmcom/db2express-c:10.5.0.5-3.10.0
+echo "su - db2inst1 -c 'db2 CONNECT TO doctrine && db2 CREATE USER TEMPORARY TABLESPACE doctrine_tbsp PAGESIZE 4 K'" > /tmp/doctrine-init.sh
+chmod +x /tmp/doctrine-init.sh
+
 sudo docker run \
     -d \
     -p 50000:50000 \
     -e DB2INST1_PASSWORD=Doctrine2018 \
     -e LICENSE=accept \
+    -e DBNAME=doctrine \
+    -v /tmp/doctrine-init.sh:/var/custom/doctrine-init.sh:ro \
     --name db2 \
-    ibmcom/db2express-c:10.5.0.5-3.10.0 \
-    db2start
-
-sleep 15
+    --privileged=true \
```

Comment:
Is it really necessary to have this flag enabled? Not a big deal but it's usually recommended to give the minimum permissions to the container.

Reply:
Yes, it's necessary.